### DISCUSSION TITLE
Allow for opt-in key repeat on keybind

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [YaLTeR/niri#577] [Ideas]
While it wouldn't be ideal for most keybinds, having the option to turn key-repeat on when holding down a keybind would be great for volume and brightness control keys :)

**Comment by YaLTeR** [comment]
Already implemented and default in 0.1.8

**Reply by ThatOneCalculator** [reply]
Oh nice, what's the config for that? Don't see it in the docs unless I missed smth

**Reply by ThatOneCalculator** [reply]
Ah! 

![image](https://github.com/user-attachments/assets/c7928ccc-a44c-410e-8fb5-7b0bf7a325c5)
